scanner/services/indexer.go

```diff
@@ -137,6 +147,7 @@ func (s *indexerService) GetOrCreateIndexReport(ctx context.Context, req *v4.Get
 		return nil, err
 	}
 
+	zlog.Info(ctx).Msg("index report for container image does not exist; creating...")
```

Comment:
super nit: for consideration - since the zlog ctx will have `GetOrCreateIndexReport` and the first log message is:

`getting index report for container image`

```suggestion
	zlog.Info(ctx).Msg("creating index report for container image")
```